### DISCUSSION TITLE
Why does Grid component have negative margin?

### DISCUSSION THREAD

**Discussion by AxelReid** [mantinedev/mantine#3130] [Q&A]
Just wondering why does the `Grid` component have a negative margin value?

**Comment by rtivital** [comment]
To offset columns gutter. If you do not want it, set `<Grid gutter={0} />`

**Reply by AxelReid** [reply]
Yea, i see it's half of the gutter value. No i want gutter.
I see no difference when removing the margin (`<Grid m={0} />`). If i don't remove that margin it is causing overlapping bug to sibling contents. It's not a problem to me just that seems to be unnecessary

**Reply by AxelReid** [reply]
My bad. I got it why it's used. `overflow: 'hidden'` to parent element is fixing the issues
